app/views/users/_spamaway.html.erb

```diff
@@ -16,7 +16,7 @@
         <div class="btn-group btn-group-justified" role="group">
           <% [0,1].each_with_index do |s, j| %><% statement = turingtest[i][s] %>
             <div class="btn-group" role="group">
-              <button type="button" class="col-xs-5 btn btn-default" style="font-size:3em;text-align:left;<% if i.odd? %> background:#eef;<% end %>" id="spamaway-<%= i.to_s + j.to_s %>">
+              <button type="button" class="col-xs-5 btn btn-default" style="font-size:3em;text-align:left;<% if i.odd? %> background:#eef;<% end %>" id="spamaway-<%= caller %>-<%= i.to_s + j.to_s %>"
```

Comment:
This also doesn't appear to be addressing the radio buttons' duplicate IDs problem as described in #4966. With this change in place, we still have 4 radio buttons with the ID `spamaway_statement1_`. 
So while this change is, in my opinion, a good progress, I suggest changing the next line so that the radio buttons also have unique IDs. Maybe something like this:
```ruby
<%= spam.radio_button vars[i], statement, { id: "spamaway_statement_#{caller}_#{i}#{j}" } %> <%= statement %>
```
I tested this locally and it seems to work, however my knowledge of this spam filter isn't the best, so it would be good to have another person take a look at it.

Reply:
Hi @alonpeer Thank you so much for your review.
I missed out the radio buttons. And The change you provided is correct.
I tested it. This would assign a unique id to all radio buttons values. Thus making them unique.
Thanks !!!
I've updated the PR with newer hash Ruby style and the change regarding radio buttons.

Reply:
The PR is updated as requested :+1: